orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -491,7 +491,12 @@ def __init__(self, parent=None):
             widget.change_signal.connect(self.settings_invalidated)
 
         frame_layout.addStretch()
-        self.mainArea.layout().addWidget(frame)
+        scroll = QtGui.QScrollArea()
```

Comment:
I don't like this. Why would we want a horizontal scroll?
<img width="742" alt="screen shot 2016-08-12 at 15 46 08" src="https://cloud.githubusercontent.com/assets/713026/17624211/0513a3ca-60a4-11e6-92eb-992adf676429.png">

I prefer the previous state, i.e.:
<img width="1142" alt="screen shot 2016-08-12 at 15 46 38" src="https://cloud.githubusercontent.com/assets/713026/17624243/22a16f9e-60a4-11e6-9598-04144c9ee491.png">

IMO horizontal scroll is very confusing. I don't even see the ON/OFF buttons any more. Further, the usage of filtering phase is very strange since I have to scroll left and right to set everything.

If I assume this was added to make the widget thiner than I propose we rather focus on the filtering phase and make that thiner. If the widgets are so wide that we need scrolling than we should probably refactor the GUI. I'm ok with the vertical scroll but I think we should not use the horizontal one.